lib/Settings/Settings.js

```diff
@@ -40,6 +40,7 @@ class Settings extends Component {
     this.sections.forEach(section => {
       const sectionRoutes = section.pages
         .filter(p => !p.perm || stripes.hasPerm(p.perm))
+        .filter(p => !p.iface || stripes.hasInterface(p.iface))
```

Comment:
I called it `interface` initially, but since [that's a reserved word in JavaScript](https://www.w3schools.com/js/js_reserved.asp) I judged it better avoided.